services/QuillLMS/app/models/feedback_history.rb

```diff
@@ -101,4 +103,8 @@ def self.batch_create(param_array)
   private def confirm_prompt_type
     self.prompt_type = DEFAULT_PROMPT_TYPE if prompt_id && !prompt_type
   end
+
+  private def anonymize_activity_session_uid
+    self.activity_session_uid = ActivitySessionFeedbackHistory.get_feedback_history_session_uid(activity_session_uid)
```

Comment:
I think we can simplify this code by making this something like this:

```ruby
self.feedback_session_uid = FeedbackSession.find_or_create_by(activity_session_uid: activity_session_uid).uid
```
and relying on `FeedbackSession` (or Postgres) to generate a `uid` on create.